connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/ClientE2ETestSuite.scala

```diff
@@ -76,7 +76,8 @@ class ClientE2ETestSuite extends RemoteSparkSession {
     assert(result(2) == 2)
   }
 
-  test("simple udf") {
+  // Ignore this test until the udf is fully implemented.
+  ignore("simple udf") {
```

Comment:
Would you mind ignore this one in a in an independent pr first?

Reply:
I created SPARK-42665 yesterday because it still test fail in 3.4.0 RC2 and be reported in the dev mail list

Reply:
cc @hvanhovell should we ignore this test first? Maven test must fail due to the function is `org.apache.spark.sql.ClientE2ETestSuite$$Lambda$XXX`,  but there is no `ClientE2ETestSuite` in server side when test using maven.